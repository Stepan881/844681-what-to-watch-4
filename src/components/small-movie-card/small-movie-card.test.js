import React from "react";
import renderer from "react-test-renderer";

import SmallMovieCard from "./small-movie-card.jsx";

import mock from "../../mocks/films.js";

const onMouseEnter = () => {};

it(`MovieCard render card`, () => {
  const card = renderer
    .create(
        <SmallMovieCard
          film={mock.films[0]}
          onMouseEnter={onMouseEnter}
        />)
    .toJSON();

  expect(card).toMatchSnapshot();
});
