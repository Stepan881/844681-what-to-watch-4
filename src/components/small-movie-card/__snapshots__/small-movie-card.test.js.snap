// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieCard render card 1`] = `
<article
  className="small-movie-card catalog__movies-card"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
>
  <video
    className="small-movie-card__image"
    height="175"
    muted={true}
    poster="img/the-grand-budapest-hotel-poster.jpg"
    width="280"
  >
    <source
      src="sintel_trailer-480p.mp4"
      type="video/mp4"
    />
  </video>
  <h3
    className="small-movie-card__title"
  >
    <a
      className="small-movie-card__link"
      href="movie-page.html"
    >
      The Grand Budapest Hotel
    </a>
  </h3>
</article>
`;
